component/appcat_apiserver.jsonnet

```
@@ -79,7 +79,7 @@ local apiserver = loadManifest('aggregated-apiserver.yaml') {
   metadata+: {
     namespace: apiserverParams.namespace,
     annotations+: {
-      'metadata.appcat.vshn.io/enabled-service-hash': vars.GetVSHNServicesObject(),
+      'metadata.appcat.vshn.io/enabled-services-hash': std.md5(std.join("-", vars.GetEnabledVSHNServiceNames())),
```

Linter is crying about `"` not being `'`
```suggestion
      'metadata.appcat.vshn.io/enabled-services-hash': std.md5(std.join('-', vars.GetEnabledVSHNServiceNames())),
```